import-export-cli/cmd/mg/listAPIs.go

```diff
@@ -0,0 +1,96 @@
+/*
+*  Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+ */
+
+package mg
+
+import (
+	"encoding/base64"
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/spf13/cobra"
+	mgImpl "github.com/wso2/product-apim-tooling/import-export-cli/impl/mg"
+	"github.com/wso2/product-apim-tooling/import-export-cli/utils"
+	"golang.org/x/crypto/ssh/terminal"
+)
+
+var (
+	listApisCmdAPIType string
+	listApisCmdLimit   string
+	listApisUsername   string
+)
+
+const listApisCmdLiteral = "apis"
+const listApisCmdShortDesc = "Display a list of all APIs or a filtered set of APIs"
+const listApisCmdLongDesc = `Display a list of all APIs or filtered by apiType using the flag --type, -t`
```

Comment:
```suggestion
const listApisCmdLongDesc = `Display a list of all the APIs or filtered set of APIs by apiType using the flag --type  (-t)`
```

Reply:
Please add the username flag

Reply:
Thanks for noticing. Including the username flag.